core/src/main/java/nl/nn/adapterframework/stream/Message.java

```diff
@@ -877,6 +874,10 @@ public long size() {
 			return (long) context.get(MessageContext.METADATA_SIZE);
 		}
 
+		if (request instanceof String) {
+			return ((String) request).length();
+		}
```

Comment:
If the length is available in the MessagContext, that should be used

Reply:
If the size of the message equal to the length of the string? 🤔
In which places does the difference matter between binary size, and character length?

Reply:
I've updated the documentation!